package/MDAnalysis/analysis/align.py

```diff
@@ -359,27 +415,141 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
     ref_coordinates = ref_atoms.positions - ref_com
     mobile_coordinates = mobile_atoms.positions - mobile_com
 
-    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates)
-
-    R, new_rmsd = rotation_matrix(mobile_coordinates, ref_coordinates, weights=weights)
+    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates, weights)
 
     if subselection is None:
-        atoms = mobile.universe.atoms
+        mobile_atoms = mobile.universe.atoms
     elif type(subselection) is str:
-        atoms = mobile.select_atoms(subselection)
+        mobile_atoms = mobile.select_atoms(subselection)
     else:
         try:
-            atoms = subselection.atoms
+            mobile_atoms = subselection.atoms
         except AttributeError:
             raise TypeError("subselection must be a selection string, a AtomGroup or Universe or None")
 
-    atoms.translate(-mobile_com)
-    atoms.rotate(R)
-    atoms.translate(ref_com)
-
+    #_fit_to DOES subtract center of mass, will provide proper min_rmsd
+    mobile_atoms, new_rmsd = _fit_to(mobile_coordinates,\
+        ref_coordinates, mobile_atoms, mobile_com, ref_com, weights=weights)
     return old_rmsd, new_rmsd
 
 
+class AlignTraj(AnalysisBase):
+    """RMS-fit trajectory to a reference structure using a selection.
+
+    Both reference *ref* and trajectory *traj* must be
+    :class:`MDAnalysis.Universe` instances. If they contain a
+    trajectory then it is used. The output file format is determined
+    by the file extension of *filename*. One can also use the same
+    universe if one wants to fit to the current frame.
+    """
+    def __init__(self, mobile, reference, select='all', filename=None, prefix='rmsfit_',
+                    mass_weighted=False, tol_mass=0.1, strict=False, force=True, quiet=False,
+                    start=None, stop=None, step = None, **kwargs):
```

Comment:
to long lines and spaces around `=` in function declaration. A autoformatter will help with this.